src/components/molecules/ArticleTrashList/index.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <div class="article-trashed">
+    <appHeading :level="1">削除済み記事一覧</appHeading>
```

Comment:
[nits]

他の記述と合わせてケバブケースで記述するようにお願いします！
他の箇所も確認してみてください！

```suggestion
    <app-heading :level="1">削除済み記事一覧</app-heading>
```

Reply:
6125134dedf6dddd6d2556f62cd12d6efa55f6b9
修正いたしました！！